opsdroid/parsers/rasanlu.py

```diff
@@ -98,6 +98,8 @@ async def has_compatible_version_rasanlu(config):
     """Check if Rasa NLU is compatible with the API we implement"""
     _LOGGER.debug(_("Checking Rasa NLU version."))
     json_object = await _get_rasa_nlu_version(config)
```

Comment:
Should the connection error be raised in this method instead?

Reply:
It is raised in `_get_rasa_nlu_version`: https://github.com/oleg-fiksel/opsdroid/blob/5aea46abb4ca292a94656ac7b41ab029fbfb4466/opsdroid/parsers/rasanlu.py#L85

and therefore returns`None`, which is checked in `has_compatible_version_rasanlu`.

Maybe there is better way to do it?